tests/test_ios_appstore_lane_identity.py

```
@@ -0,0 +1,345 @@
+        "IOS_APPSTORE_BUNDLE_ID",
+        "IOS_APPSTORE_BUNDLE_IDENTIFIER",
+    ):
+        env.pop(key, None)
+    env["CMUX_FAKE_XCODEBUILD_LOG"] = str(tmp / "xcodebuild.jsonl")
+    env["CMUX_FAKE_EXPORT_OPTIONS_COPY"] = str(tmp / "ExportOptions.plist")
+    env["CMUX_FAKE_ASC_LOG"] = str(tmp / "asc.jsonl")
+    env["IOS_DISTRIBUTION_IDENTITY"] = IDENTITY
+    return env
+
+
+def _run(args: list[str], *, env: dict[str, str], tmp: Path) -> subprocess.CompletedProcess[str]:
```

Ruff (0.15.20)

[warning] Unused function argument: `tmp`

(ARG001)